SPECS/curl.spec

```diff
@@ -503,6 +493,41 @@ rm -f ${RPM_BUILD_ROOT}%{_mandir}/man1/mk-ca-bundle.1*
 %endif
 
 %changelog
+* Mon Apr 28 2025 Thierry Escande <thierry.escande@vates.tech> - 8.9.1-5.1
+- Update to curl v8.9.1 from el10 srpm
+- Restore changes from original fc40 specfile
```

Comment:
Suggestion for both lines as one: Restore changes from original fc40 specfile then update to curl-8.9.1-5.el10

Reply:
I still suggest this. The current changelog gives a biased view of the changes, IMO.